conf/any_slurm.config

```diff
@@ -8,12 +8,7 @@ vim: syntax=groovy
  */
 
 
-if (params.clusterOptions == [:]){
-    throw new RuntimeException('You must specify --clusterOptions. For instance "-A <your_slurm_account>". Use an empty string if no options are applicable for your system.')
-}
-else {
-    process.clusterOptions = params.clusterOptions
-}
+process.clusterOptions = params.clusterOptions
```

Comment:
Do we need this ? I guess not having this line won't make any difference :)

Reply:
I'm not sure 🤷‍♂️

Reply:
hehe.. me neither. Anyway it doesn't hurt, so I will merge 👍

Reply:
Oh it was merged already 😜

Reply:
💨